content/300-guides/300-upgrade-guides/200-upgrading-versions/600-upgrading-to-prisma-5.mdx

```diff
@@ -0,0 +1,321 @@
+---
+title: 'Upgrade to Prisma 5'
+metaTitle: 'Upgrade to Prisma 5'
+metaDescription: 'Guides on how to upgrade to Prisma 5'
+tocDepth: 3
+toc: true
+---
+
+<TopBlock>
+
+Prisma 5.0.0 introduces a number of **breaking changes**. This guide explains how upgrading might affect your application and gives instructions on how to handle any breaking changes.
+
+</TopBlock>
+
+## Major changes
```

Comment:
@janpio and I had a back and forth on this. The thinking on "Major" and "Minor" vs "Breaking" is that all the changes in this guide can be breaking. Some are just more disruptive.

Reply:
I think we got rid of "Major" now to avoid mixing it up with "major release" etc.
We now have something that expresses that there are some bigger and more common, vs smaller or obscure changes.